swagger.yaml

```diff
@@ -787,7 +787,7 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
-  /clusters/{cluster_id}/discovery-ignition:
+  /v1/clusters/{cluster_id}/discovery-ignition:
```

Comment:
this should also be based on infra_env, no?

Reply:
I don't think so, v1 should remain as it is now,